sky/engine/core/css/FontSize.cpp

```diff
@@ -53,7 +53,7 @@ float FontSize::getComputedSizeFromSpecifiedSize(const Document* document, bool
     // However we always allow the page to set an explicit pixel size that is smaller,
     // since sites will mis-render otherwise (e.g., http://www.gamespot.com with a 9px minimum).
 
-    Settings* settings = document->settings();
+    Settings* settings = document ? document->settings() : nullptr;
     if (!settings)
```

Comment:
We'll probably delete a lot of this code once we cut over to using paragraph builder.